src/flamenco/txn/fd_txn_generate.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * Provides utility methods to create txn templates for
+ * pre-staging, as well as a mechanism to build out an 
+ * entire transaction with instructions.
+*/
+
+#include "../../ballet/txn/fd_txn.h"
+#include "../../ballet/txn/fd_compact_u16.h"
+#include "../../flamenco/types/fd_types_custom.h"
+#include "../../flamenco/types/fd_types.h"
+
+#define SIGNATURE_SZ 64
+#define ACCOUNT_SZ   32
+#define BLOCKHASH_SZ 32
```

Comment:
You should probably use the pre-existing definitions for these, but if not, they definitely need to be prefixed, since they're #defines in a header.